config/locales/es.yml

```diff
@@ -122,6 +117,7 @@ es:
   any_special_requests: '¿Cuáles son sus peticiones especiales (como silla de ruedas, asientos de bebé)? O responder ninguno.'
   thanks_will_contact: '¡Gracias! Tenemos su información y nos pondremos en contacto con usted antes de su viaje.'
   thanks_wait_for_driver: '¡Gracias! Tenemos su información y nos pondremos en contacto con usted cuando nos encontramos con un conductor para usted.'
+  auto_confirmed_wait_for_driver: "Su viaje is scheduled for %{time}. You don't need to do anything. Responda 4 para hablar con alguien"
```

Comment:
Not fully translated. Perhaps
```
"Su viaje está programado para las %{time}. No necesitas hacer nada. Responda 4 para hablar con alguien."
```
though I'd suggest a native speaker refine that.

Reply:
A native (Mexican) Spanish speaker suggests
```
"Tu viaje con Drive the Vote está programado para %{time}. No necesitas hacer nada más. Responde 4 para hablar con alguien."
```

Reply:
ha. yes. best effort spanglish. i'll solicit a native speaker on slack